packages/core/src/storage/migration/updates/0.5-0.6/migrateToCredoFolder.ts

```diff
@@ -0,0 +1,36 @@
+import type { BaseAgent } from '../../../../agent/BaseAgent'
+import type { FileSystem } from '../../../FileSystem'
+
+import { InjectionSymbols } from '../../../../constants'
+import { CredoError } from '../../../../error'
+
+/**
+ * Migrates the sqlite folder location from .afj to .credo in the storage directory in node and react native.
+ *
+ */
+export async function migrateToCredoFolder<Agent extends BaseAgent>(agent: Agent) {
```

Comment:
due to the sensivity in doing migrations it's always good to test these migrations extensively. For all other migrations and version changes we have tests (per migration method, and for a version migration in general), can you add these?